config/flp/flowlogs-pipeline.conf.yaml

```diff
@@ -0,0 +1,124 @@
+# This file was generated automatically by flowlogs-pipeline confgenerator
```

Comment:
please remove this file

Reply:
We need to save the file somewhere so we can then embed it in the code. I moved it to the location of the code: controllers/flowlogspipeline/flowlogs-pipelin-metricse.conf.yaml

Reply:
@KalmanMeth my mistake, please  put the file back in the `config/flp/flowlogs-pipeline.conf.yaml` it is needed and a good place for this file.